IFC4x3/Sections/Resource definition data schemas/Schemas/IfcExternalReferenceResource/Entities/IfcDatasetReference/DocEntity.xml

```diff
@@ -4,7 +4,9 @@
 		<DocAttribute Name="Description" UniqueId="ca30fa31-ac3e-4469-ae6f-26d92e6ff6b2" DefinedType="IfcText" AttributeFlags="1">
 			<Documentation>Description of the dataset reference for informational purposes.</Documentation>
 		</DocAttribute>
-		<DocAttribute Name="ReferencedDataset" UniqueId="b967d95b-2e3b-44f4-9251-a812c28ce43c" DefinedType="IfcDatasetInformation" AttributeFlags="1" />
+		<DocAttribute Name="ReferencedDataset" UniqueId="b967d95b-2e3b-44f4-9251-a812c28ce43c" DefinedType="IfcDatasetInformation" AttributeFlags="1">
+			<Documentation>The dataset that is referenced.</Documentation>
```

Comment:
Same as on info. Probably these two should just be switched, right?

Reply:
> Same as on info. Probably these two should just be switched, right?

No, I think that this one is correct and in line with IfcDocumentReference.ReferencedDocument.

Reply:
Updated description for both IfcDatasetReference and IfcDocumentReference in [39a07c2](https://github.com/bSI-InfraRoom/IFC-Specification/pull/869/commits/39a07c27fb24abced66e01e611f9230ab1d73764)